sound/soc/sof/pm.c

```diff
@@ -500,3 +500,58 @@ void snd_sof_d0i3_entry_work(struct work_struct *work)
 	}
 }
 EXPORT_SYMBOL(snd_sof_d0i3_entry_work);
+
+/**
+ * sof_set_dsp_d0i0 - bring ADSP back to d0i0
+ * @force: force to exit from d0i3
+ * @reload: reload unused pipelines/widgets into FW
+ */
+void snd_sof_set_dsp_d0i0(struct snd_sof_dev *sdev, bool force, bool reload)
+{
+	int ret;
+
+	if (sdev->d0_state == SOF_DSP_D0I0) {
+		dev_vdbg(sdev->dev, "Already in D0i0.\n");
+		return;
+	}
+
+	if (sof_d0ix_target_state(sdev) == SOF_DSP_D0I3 &&
+	    !force) {
```

Comment:
cannot the above two lines be joined? That doesn't seem to exceed 80 characters.